builder/triton/access_config.go

```diff
@@ -100,7 +100,7 @@ func (c *AccessConfig) createSSHAgentSigner() (authentication.Signer, error) {
 	}
 
 	// Ensure we can sign a request
-	_, err = signer.Sign("Wed, 26 Apr 2017 16:01:11 UTC")
+	_, err = signer.Sign("Wed, 26 Apr 2017 16:01:11 UTC", false)
```

Comment:
That's an oooold breaking change from  https://github.com/joyent/triton-go/pull/135